### DISCUSSION TITLE
Using backrest with authelia and traefik, cant login

### DISCUSSION THREAD

**Discussion by trexxeon** [garethgeorge/backrest#146] [Q&A]
Hello,

I am trying to protect the backrest ui with authelia, and I am using traefik as proxy.

I add backrest domain as authelia protected site, which works. I can login through authelia, I am then greeted with the backrest page. And when I try to log in to that page I get the "loggen in" alert, but quickly I am facing the login page again.
In the logs I see 

	DEBUG	login request	{"username": "admin"}
	DEBUG	login request	{"username": "default"}
	 WARN	failed login attempt	{"error": "user not found")
admin is the user  I am trying to login with, so I don't know why it says that there's a login request from "default"

In the browser console I get 201 and "No authorization header"

anyone got any ideas?

**Comment by garethgeorge** [comment]
The login request for "default" is backrest checking if you are a first time user, "default" is the default username if none has been created yet.

Re: your auth problems, is authelia possibly stripping the Authorization header from the request?

**Reply by trexxeon** [reply]
Okey, yes it probably is authelia that strips the authorization header. Without authelia involved it works.

Is it possible to skip the login for backrest? since authelia is able to use mfa I'd rather use that only.

**Reply by garethgeorge** [reply]
Hey, a few other questions touch on this but incase you haven't found a workaround yet you can disable auth by creating a user with the username "default" and password "password". Backrest will automatically login as this user on first connection (it's also the user used for initial setup).

I've added a page to the wiki that talks a bit about how to get this setup.
